cms/templates/import.html

```diff
@@ -127,6 +185,19 @@ <h3 class="title">${_("Success")}</h3>
       </form>
     </article>
 
+  %if library:
+    <aside class="content-supplementary" role="complimentary">
+      <div class="bit">
+          <h3 class="title-3">${_("Why import a library?")}</h3>
+          <p>${_("You may want to run a new version of an existing library, or replace an existing library altogether. Or, you may have developed a library outside Studio.")}</p>
+      </div>
+
+      <div class="bit">
+          <h3 class="title-3">${_("Warning: Importing while a library is in use")}</h3>
+          <p>${_("If you perform an import while your library is in use, and you change the URL names (or url_name nodes) of any Problem components, the student data associated with those Problem components may be lost. This data includes students' problem scores.")}</p>
```

Comment:
@Kelketek @antoviaque @marcotuts Can we delete this warning completely? It's not relevant to libraries, I don't think.
Is there any alternative information/warning that we should provide? Problems inside randomized content blocks that come from a library that has been overwritten will be out of sync, is that correct? So perhaps we replace the warning with something about re-syncing problems to get the up-to-date version in the library? Please confirm, thanks!